modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -394,6 +395,28 @@ public PartitionReplicaListener(
         prepareIndexBuilderTxRwOperationTracker();
     }
 
+    private void runPersistentStorageScan() {
+        try (Cursor<IgniteBiTuple<UUID, TxMeta>> txs = txStateStorage.scan()) {
+            for (IgniteBiTuple<UUID, TxMeta> tx : txs) {
+                UUID txId = tx.getKey();
+                TxMeta txMeta = tx.getValue();
+
+                assert !txMeta.enlistedPartitions().isEmpty();
+
+                if (isFinalState(txMeta.txState())) {
+                    txManager.cleanup(
```

Comment:
Let's handle the future in order to log the expcetion if any. I understand that we really do not expect ones because of durable cleanup, however simple no-op may hide the problem.